eventhubs/data-plane/pom.xml

```diff
@@ -1,79 +1,84 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-	<description>Java libraries for talking to Windows Azure Event Hubs</description>
-	
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+							 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	
 	<groupId>com.microsoft.azure</groupId>
 	<artifactId>azure-eventhubs-clients</artifactId>
-	<version>2.0.0</version>
 	<packaging>pom</packaging>
-	
+	<version>2.0.0</version>
+
+	<name>Microsoft Azure Event Hubs SDK Parent</name>
+	<description>Java libraries for talking to Windows Azure Event Hubs</description>
 	<url>https://github.com/Azure/azure-event-hubs</url>
-	
+
+	<parent>
+		<groupId>com.azure</groupId>
+		<artifactId>azure-client-sdk-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+		<relativePath>../../pom.client.xml</relativePath>
+	</parent>
+
+	<distributionManagement>
+		<site>
+			<id>azure-java-build-docs</id>
+			<url>${site.url}/site/${project.artifactId}</url>
+		</site>
+	</distributionManagement>
+
+	<scm><url>https://github.com/Azure/azure-sdk-for-java</url></scm>
+
 	<properties>
 		<proton-j-version>0.31.0</proton-j-version>
 		<qpid-proton-j-extensions-version>1.1.0</qpid-proton-j-extensions-version>
-	  	<junit-version>4.12</junit-version>
-		<slf4j-version>1.8.0-alpha2</slf4j-version>
 	</properties>
-	
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.5</version>
-				<configuration>
-					<source>1.8</source>
-					<target>1.8</target>
-					<optimize>true</optimize>
-					<showDeprecation>true</showDeprecation>
-					<showWarnings>true</showWarnings>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
+
 	<repositories>
 		<repository>
 			<id>HTTP proxy server in java</id>
 			<url>http://raw.github.com/SreeramGarlapati/jproxy/master/releases</url>
 		</repository>
 	</repositories>
```

Comment:
Is this repositories section necessary? It doesn't appear to resolve in a browser, so I'm guessing it can probably be removed?

Reply:
I am not sure about this part. It will failed the mvn install when it is deleted.

[ERROR] Failed to execute goal on project azure-eventhubs-clients: Could not resolve dependencies for project com.microsoft.azure:azure-eventhubs-clients:pom:2.0.0: Failure to find org.jutils.jproxy:jproxy:jar:0.0.1 in https://oss.sonatype.org/content/repositories/snapshots/ was cached in the local repository, resolution will not be reattempted until the update interval of ossrh has elapsed or updates are forced -> [Help 1]